Vagrantfile

```diff
@@ -39,13 +39,15 @@ else
   VM_CPUS      = ENV['VM_CPUS']      || '4'                                # Amount of CPU cores for DEV VM
   VM_NAME      = ENV['VM_NAME']      || "Spryker Dev VM (#{VM_PROJECT})"   # Visible name in VirtualBox
   VM_SKIP_SF   = ENV['VM_SKIP_SF']   || '0'                                # Don't mount shared folders
+  VM_STORES    = (ENV['VM_STORES'] || 'de,at,us').split(',')               # Hostnames to be managed
 
   config=
     "VM_PROJECT =         '#{VM_PROJECT}'\n" +
     "VM_IP =              '#{VM_IP}'\n" +
     "VM_MEMORY =          '#{VM_MEMORY}'\n" +
     "VM_CPUS =            '#{VM_CPUS}'\n" +
     "VM_NAME =            '#{VM_NAME}'\n" +
+    "VM_STORES =          '#{VM_STORES}'\n" +
```

Comment:
This will produce array in quotes, `VM_STORES = '["de", "at", "us"]'` - please remove surrounding quotes (same for `Vagrantfile-quick`

Reply:
Oh, sorry I couldn't really test it, thank you for catching that.